shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/di/koin/modules/ViewModelModule.kt

```diff
@@ -1,8 +1,10 @@
 package co.nimblehq.blisskmmic.di.koin.modules
 
 import co.nimblehq.blisskmmic.presentation.modules.login.LoginViewModel
+import co.nimblehq.blisskmmic.presentation.modules.resetpassword.ResetPasswordViewModel
 import org.koin.dsl.module
 
 val viewModelModule = module {
     single<LoginViewModel> { LoginViewModel(get()) }
```

Comment:
I prefer to use [Constructor DSL](https://insert-koin.io/docs/reference/koin-core/dsl-update), you don't need to update this file even you add more use cases into the view model.

```kotlin
viewModelOf(::LoginViewModel)
viewModelOf(::ResetPasswordViewModel)

// or
singleOf(::LoginViewModel)
singleOf(::ResetPasswordViewModel)
```

Also, please double check about `viewModelOf` or `singleOf`, which one is right?